cmd/gcp-controller-manager/node_csr_approver.go

```diff
@@ -695,6 +697,45 @@ func validateInstanceGroupHint(instanceGroupUrls []string, instanceGroupHint str
 	return resolved, nil
 }
 
+var errNotFoundListReferrers = errors.New("not found the entry in ListReferrers")
+
+func checkInstanceReferrersBackOff(ctx *controllerContext, instance *compute.Instance, clusterInstanceGroupUrls []string) (bool, error) {
```

Comment:
nit: this function is never really returning a real erro. Maybe we should only return bool.

Reply:
done.